build/cmake/global_compiler_flags.cmake

```
@@ -139,6 +139,14 @@ if(IS_MSVC)
     add_compile_definitions(UNICODE _UNICODE)
 endif()
 
+# Ensure that debug WASM builds include additional WASM-specific linker flags 
+if (IS_WASM AND CMAKE_BUILD_TYPE STREQUAL "Debug")
+    string(APPEND CMAKE_EXE_LINKER_FLAGS " -sASSERTIONS=2 -sSAFE_HEAP -sDEMANGLE_SUPPORT")
```

```suggestion
    string(APPEND CMAKE_EXE_LINKER_FLAGS " -sASSERTIONS=2 -sSAFE_HEAP")
```

> em++: warning: DEMANGLE_SUPPORT is deprecated (mangled names no longer appear in stack traces). Please open a bug if you have a continuing need for this setting [-Wdeprecated]

Looks like they started reporting unmangled names by default back in Feb or March.